docs/scripts_md/File.md

```diff
@@ -110,6 +110,15 @@ Gets the database handle reference which the object is using internally.
 
 RETURNS: DBI database handle reference.
 
+### getFileType($file)
+
+Determines the imaging file type based on the extension of the file to insert
+and the list of available types in the ImagingFileTypes table of the database.
```

Comment:
small minor comment:
can the table be encapsulated in `C<>`?

Reply:
@MounaSafiHarab oops. Resolved :)